scripts/package/postinstall.sh

```diff
@@ -56,9 +60,103 @@ EOF
 }
 
 manage_sysv_service() {
+  # Ensure systemd script isn't present, and if it is remove it
+  if [ -f /usr/lib/systemd/system/observiq-otel-collector.service ]; then
+    rm -f /usr/lib/systemd/system/observiq-otel-collector.service
+  fi
+  # Ensure aix env vars file isn't present, and if it is remove it
+  if [ -f /opt/observiq-otel-collector/observiq-otel-collector.aix.env ]; then
+    rm -f /opt/observiq-otel-collector/observiq-otel-collector.aix.env
+  fi
+
   chmod 755 /etc/init.d/observiq-otel-collector
   chmod 644 /etc/sysconfig/observiq-otel-collector
   echo "configured sysv service"
+
+  cat << EOF
+
+The "observiq-otel-collector" service has been configured!
+
+The collector's config file can be found here: 
+  /opt/observiq-otel-collector/config.yaml
+
+To view logs from the collector, run:
+  sudo tail -F /opt/observiq-otel-collector/log/collector.log
+
+For more information on configuring the collector, see the docs:
+  https://github.com/observIQ/bindplane-agent/tree/main#observiq-opentelemetry-collector
+
+To stop the observiq-otel-collector service, run:
+  sudo service observiq-otel-collector stop
+
+To start the observiq-otel-collector service, run:
+  sudo service observiq-otel-collector start
+
+To restart the observiq-otel-collector service, run:
+  sudo service observiq-otel-collector restart
+
+To enable the service on startup, run:
+  sudo service observiq-otel-collector enable
+
+If you have any other questions please contact us at support@observiq.com
+EOF
```

Comment:
These seems unnecessary to me. The install script includes this output already.  I think we could consider moving it here but its not specific to AIX.

Reply:
I duplicated what already existed in this script for the other init forms.

Reply:
Additionally, this isn't output from this script when using the install_unix.sh. It is only shown when rpm is used manually.